### DISCUSSION TITLE
Livarno / Lidl Smart LED Ceiling Light 14153806L

### DISCUSSION THREAD

**Discussion by bjoerns1983** [Koenkk/zigbee2mqtt#12101] [Devices]
I have just received a new Livarno / Lidl Smart LED Ceiling  Light. 
https://www.lidl.de/p/livarno-home-deckenleuchte-rgb-mit-backlightfunktion-fur-zigbee-smart-home-best/p100335196002
It has the Product Number 14153806L and is identified by Zigbee2MQTT as Tuya TS0504B, with Zigbee Vendor     _TZ3210_sroezl0s

It works as expected, i just liked to know how to add it to the Zigbee2MQTT Device Database.

**Comment by Koenkk** [comment]
Could you provide the color temp range? Ssee https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html#_3-adding-converter-s-for-your-device

**Reply by bjoerns1983** [reply]
Sure:

Read result of 'lightingColorCtrl': {"colorTempPhysicalMin":153,"colorTempPhysicalMax":500}

**Reply by Koenkk** [reply]
Added.

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Reply by bjoerns1983** [reply]
Thank you!

**Reply by NSURLSession0** [reply]
@bjoerns1983 @Koenkk Is changing the color temperature (still?) working for you? I received the  Lidl 14153806L ceiling light today and changing the color temperature in HA doesn't seem to do anything. I can only change the brightness and the light colors.

**Reply by chris400** [reply]
@Mauricevb see https://github.com/Koenkk/zigbee2mqtt/issues/14439
